CONTRIBUTING.md

```diff
@@ -77,18 +110,18 @@ Once `pyenv` and `pyenv-virtualenv` are installed on your system, you
 can create and configure the Python virtual environment with these
 commands:
 
-```bash
+```console
 cd skeleton-generic
 pyenv virtualenv <python_version_to_use> skeleton-generic
 pyenv local skeleton-generic
```

Comment:
This never got updated from the skeleton.

Reply:
Sharp eyes!  Fixed in commit a6d2e5e.